src/domain/space/space.defaults/space.defaults.service.ts

```
@@ -66,21 +66,26 @@ export class SpaceDefaultsService {
     }
 
     // Enforce innovation flow settings:
-    const maxNumberOfStates =
-      templateSpaceContent.collaboration?.innovationFlow?.settings
-        .maximumNumberOfStates ?? Number.MAX_SAFE_INTEGER;
-    const minNumberOfStates =
-      templateSpaceContent.collaboration?.innovationFlow?.settings
-        .minimumNumberOfStates ?? 0;
+    if (templateSpaceContent.collaboration?.innovationFlow?.settings) {
```

if the settings are missing I would raise an error here